lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1311,6 +1347,22 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
                         chain=certificate["chain"],
                     )
 
+    def _on_relation_broken(self, event: RelationBrokenEvent) -> None:
+        """Handler triggered on relation broken event.
+
```

Comment:
How about adding to the docstring here? Here's my understanding.
Feel free to ignore if it's too trivial.

```suggestion

        Emitting `all_certificates_invalidated` from `relation-broken` rather
        than `relation-departed` since certs are stored in app data.
```

Reply:
Agree, an old me would have appreciated it. Added.